src/main/kotlin/no/skatteetaten/aurora/databasehotel/web/rest/DatabaseInstanceController.kt

```diff
@@ -16,7 +16,8 @@ data class DatabaseInstanceResource(
     val host: String,
     val createSchemaAllowed: Boolean,
     val instanceName: String,
-    val port: Int
+    val port: Int,
+    val affiliation: String?
```

Comment:
Modellen som jeg har brukt i dbh til nå er at den skal være helt frikoblet fra plattformbegreper direkte (slik som f.eks. affiliation). Det er derfor vi benytter labels, slik at vi gjør det mulig å løst knytte en instans til en affiliation, men domeneforståelsen av affiliation ligger utenfor dbh. Jeg foreslår at dere eksponerer hele instanceLabels-mappen direkte her, og dersom man ønsker knytte en affiliation-property til en databaseinstans så gjøres dette i gobo.

Reply:
takk for innspill, da endrer jeg dette